rheem-core/src/main/java/org/qcri/rheem/core/function/ConsumerDescriptor.java

```diff
@@ -0,0 +1,94 @@
+package org.qcri.rheem.core.function;
```

Comment:
I fail to understand what does this class represent? Why do we need a separate descriptors for the sink operators ?